util/util.go

```diff
@@ -126,7 +124,7 @@ func GetServiceIDByPostmanCollectionID(c rest.FrontClient, collectionID string)
 		svcCollectionID := strings.ToLower(svc.PostmanMetaData.CollectionID)
 		svcEnv := strings.ToUpper(svc.PostmanMetaData.Environment)
 
-		if strings.EqualFold(collectionID, svcCollectionID) && strings.EqualFold(svcEnv, env) {
+		if strings.EqualFold(collectionID, svcCollectionID) {
 			result = svc.ID
 			postmanCollectionIDCache.Set(svcEnv+"|"+svcCollectionID, svc.ID, cache.DefaultExpiration)
```

Comment:
It seems like the cache key should be consistent with what we're checking.

That looks like it was an existing bug, but let's see if we can fix it up.

Reply:
Agree, this is a bug. Removed env from the cache also